lib/controllers/chapter_controller.dart

```diff
@@ -95,12 +113,15 @@ class ChapterStateController extends Controller<ChapterState> {
 final initialChapterState = Provider.autoDispose<ChapterState>((ref) {
   final books = ref.watch(booksProvider(Testaments.allBooks)).asData?.value;
   if (books != null) {
```

Comment:
i think reads would be fine in this file, instead of watch. I don't want anything rebuilding